### PR TITLE
update whatsnewdialog

### PR DESCRIPTION
**Pull request by LWinterberg**
Resolves: https://github.com/audacity/audacity/issues/7498

Reviewer: Verify that the new strings (even the #if 0'd ones) are in the translation catalogue

QA: Verify the what's new dialog - particularly the sizing (should be slightly larger than before), and that all links still work

### CODE REVIEW

File: src/WhatsNewDialog.cpp [hunk 1]
```diff
@@ -127,11 +165,8 @@ END_EVENT_TABLE()
 WhatsNewDialog::WhatsNewDialog(wxWindow* parent, wxWindowID id)
    : wxDialogWrapper(parent, id, XO("Welcome to Audacity!"))
 {
-#if defined(__WXOSX__)
-   SetSize(WindowWidth, 400);
-#else
-   SetSize(FromDIP(wxSize(WindowWidth, 430)));
-#endif
+
+   SetSize(FromDIP(wxSize(WindowWidth, WindowHeight)));
```
Comment (LWinterberg) [inline src/WhatsNewDialog.cpp:169]: I just noticed that I have perhaps a logic error here, given that I set the pixel size separately above. Please verify the sizing on macoS

Comment (kryksyh) [reply src/WhatsNewDialog.cpp:169]: Yep, as it is right now, the window is too short, and scrollbars are shown:

image

using 460px looks good

image

File: src/WhatsNewDialog.cpp [hunk 2]
```diff
@@ -39,13 +39,21 @@ namespace
...
 #else
-   constexpr auto WindowWidth = 400;
+   constexpr auto WindowWidth = 440;
+#endif
+
+#if defined(__WXOSX__)
+   constexpr auto WindowHeight = 400;
+#else
+   constexpr auto WindowHeight = 460;
 #endif
```
Comment (kryksyh) [inline src/WhatsNewDialog.cpp:57]:
```suggestion
constexpr auto WindowHeight = 460;
```

Comment (kryksyh) [reply src/WhatsNewDialog.cpp:57]: it is also a bit too short on windows

image

Comment (kryksyh) [reply src/WhatsNewDialog.cpp:57]: Nevermind, it fits okay, when the version string does not have `alpha` suffix. I believe that adjusting the dialog height automatically depending on the textbox size is not a trivial task with wxWidgets.

**Review by kryksyh — APPROVED** [review]

**Pull request merged**